autogluon/utils/tabular/ml/models/catboost/catboost_model.py

```diff
@@ -52,6 +53,16 @@ def preprocess(self, X):
     def _fit(self, X_train, Y_train, X_test=None, Y_test=None, time_limit=None, **kwargs):
         try_import_catboost()
         from catboost import CatBoostClassifier, CatBoostRegressor, Pool
+        if self.problem_type == SOFTCLASS:
+            try_import_catboostdev()  # Need to first import catboost then catboost_dev not vice-versa.
```

Comment:
yes I didn't even add catboost-dev to the CI dependencies since (multiclass) distillation is probably a niche feature and it's better IMO if CI reflects the bulk of users' experience.